tests/e2e/inversify.types.ts

```diff
@@ -66,7 +66,7 @@ const CLASSES = {
     GitOauthAppsSettings: 'GitOauthAppsSettings',
     AnimationChecker: 'AnimationChecker',
     Sanitizer: 'Sanitizer',
-    NavigationBar: 'NavigationBar'
+    NavigationBar: 'NavigationBar',
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.

Reply:
fixed with the linter code change